deps/v8/src/objects/map.cc

```diff
@@ -646,7 +646,7 @@ Map Map::FindRootMap(Isolate* isolate) const {
       // Initial map always owns descriptors and doesn't have unused entries
       // in the descriptor array.
       DCHECK(result.owns_descriptors());
-      DCHECK_EQ(result.NumberOfOwnDescriptors(),
+      DCHECK_LE(result.NumberOfOwnDescriptors(),
```

Comment:
This line had a conflict (the context above it has changed) - I only changed the affected line.